blathering-barnacles/Bangazon
# Dw ticket15 toggle programs

**Description:**
Added the ability to toggle between viewing the upcoming training programs and the past training programs. This was done by editing the original trainingList method in trainingProgram_views.py to only show upcoming events. This was done by importing timezone from django.utils and checking whether the startDate (key in Training Program table) was greater than or equal(gte) to the current time. Then I wrote a new method that was nearly identical except it used less than or equal to (lte) as a comparison so that only past programs would be shown. I created a new html page for the past programs and linked the buttons between the two pages (upcoming programs and past programs).

**Steps to Test:**

- go to localhost:8000/workforce/training

- you can click each program and check the details to make sure that the program hasn't occurred already if you'd like.

- click the "view past training programs" button at the bottom and you will see the past programs with the start date and end date and you can verify that they are past events.

- after verifying, you can click "go back to training programs" and go back to the upcoming programs.


**Link to Feature Ticket:**
https://github.com/blathering-barnacles/Bangazon/issues/15#issue-403244834
